src/components/application_manager/include/application_manager/rpc_handler_impl.h

```diff
@@ -181,8 +181,11 @@ class RPCHandlerImpl : public RPCHandler,
                           const bool validate_params = true);
   std::shared_ptr<Message> ConvertRawMsgToMessage(
       const ::protocol_handler::RawMessagePtr message);
+
   hmi_apis::HMI_API& hmi_so_factory();
   mobile_apis::MOBILE_API& mobile_so_factory();
+  ns_smart_device_link_rpc::V1::v4_protocol_v1_2_no_extra&
+  v4_protocol_so_factory();
```

Comment:
function is only called internally, i think it would be better and less convoluted to just call `app_manager_.v4_protocol_so_factory()` instead of adding another function.

Reply:
Done in 4c89818